### PR TITLE
feat(movie): add movie model and tests

### PR DESCRIPTION
**Pull request by kienan**
**What:** Adds database configuration and sets up a movies model.

**Why:** We need to set up our database models before we can configure an endpoint.

**Details:**
- Updates config files with DB connection
- Sets up `Bookshelf` and `Knex`
- Adds a migration for movies table
- Defines a movie model
- Adds test for the movie model

### CODE REVIEW

**Review by dckrooney — APPROVED** [review]
LGTM

**Review by sjl2 — APPROVED** [review]
LGTM

File: lib/models/movie.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
```
Comment (kienan) [inline lib/models/movie.js:7]: Why is this not ES6-ified?

Comment (kienan) [reply lib/models/movie.js:7]: Talked IRL -- has to do with the way `this` works in arrow functions

**Pull request merged**